addons/storysource/package.json

```diff
@@ -35,7 +35,8 @@
     "regenerator-runtime": "^0.12.1"
   },
   "peerDependencies": {
-    "react": "*"
+    "react": "*",
+    "@storybook/source-loader": "*"
```

Comment:
we will ask the developer to pull the source-loader separately

Reply:
@shilman @ndelangrn should we add this line to our docs? 

> Please don't use caret when depending on Storybook packages, as we regularly do breaking changes in minor releases